packages/api/src/util/ensemble.js

```diff
@@ -12,3 +12,39 @@
  *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
  */
+
+const axios = require('axios');
+const { ensembleAPI } = require('../endPoints');
+
+/**
+ * Sends a request to the Ensemble API for an organization to claim sensors
+ * @param {Response} res - The HTTP response object.
+ * @param {uuid} organizationId - a uuid for an Ensemble organization
+ * @param {Array} esids - an array of ids for Ensemble devices
+ * @param {String} accessToken - a JWT token for accessing the Ensemble API
+ * @returns {Object} - the response from the Ensemble API
+ * @async
+ */
+async function bulkSensorClaim(res, organizationId, esids, accessToken) {
+  try {
+    return await axios.post(
```

Comment:
could you have this return an object containing the `success`, `does_not_exist`, `already_owned`, and `occupied` lists instead of just the API response? I feel like that would be easier to work with elsewhere in the codebase